### ISSUE TITLE
Nvim-R-df-view doesn't work in (normal) vim

### ISSUE THREAD

**Issue by protist** [jalvesaq/Nvim-R#74]
In stock vim (not neovim), I try to inspect a data frame with `<LocalLeader>rv`. I get the following message.
```
Wait...
Neovim client not running.
Press ENTER or type command to continue
```
Nothing else happens.

* Up-to-date Arch Linux
* vim 7.4.1797
* nvim-r 0.9.3
* r 3.3.0

**Comment by jalvesaq** [comment]
Here, it works with both Vim (7.4.1907) and Neovim (current development version). The last released version of Nvim-R was 0.9.5. You should update Nvim-R and either update Vim or install Neovim.

I am sorry, but I can't fix bugs in released versions. I can only make changes in the development version of the plugin which follows the development versions of both Vim and Neovim.

**Comment by protist** [comment]
Okay, that makes perfect sense. Unfortunately, Arch had the newest vim held up in testing for about a month (shocking, I know!). Meanwhile, the current version of vim wasn't qualified for nvim-r. Anyway, I just installed the latest vim from Arch's "testing" repository, along with the latest nvim-r, and now nvim-r-df-view works perfectly. Thanks for the comments and apologies for the noise.
